docs/getting-started/faq.md

```diff
@@ -0,0 +1,17 @@
+# Fervently Anticipated Questions (FAQ)
```

Comment:
🤣  Ahahaha I love this but let's keep it straightforward for non-native english speakers

```suggestion
# Frequently Asked Questions (FAQ)
```